hardware/asic/skywater/Makefile

```diff
@@ -80,10 +83,10 @@ clean_sram:
 
 clean_asic:
 	rm -rf $(OPENLANE_DESIGNS)/system
-	rm -rf iob_dp_ram.v
+	rm -rf iob_sp_ram_be.v
 	rm -rf $(OPENLANE_HOME)/run_interactive.tcl
-clean_iob_dp_ram:
```

Comment:
I don't understand why have different targets for clean memory wrapper files. Why not clean them in clean_asic target?

Reply:
I will make one target

Reply:
I will revert the config.mk file changes